src/components/button/bl-button.ts

```diff
@@ -50,6 +50,18 @@ export default class BlButton extends LitElement {
   @property({ type: String })
   label: string;
 
+  /**
+   * Sets the button label for loading status.
+   */
+  @property({ type: String, attribute: 'loading-label' })
+  loadingLabel: string;
+
+   /**
+   * Sets loading state of button
+   */
+   @property({ type: Boolean })
+   loading = false;
```

Comment:
add `reflect: true` to parameters, since this is used in style queries.

And I think there is a problem with indentation here.